pages/services/edge-lb/0.1.9/release-notes/index.md

```diff
@@ -29,6 +29,7 @@ Shortlist:
 
 ## Known Limitations
 
+* The Edge-LB v0.1.9 CLI requires that all pool definitions are contained within a single YAML or JSON file. Pools cannot be configured independently because of this limitation.
```

Comment:
Do we need to highlight that we have removed this limitation from 1.0?

Reply:
In the 1.0.0 noteworthy changes, we have this line: `- New CLI command layout, ability to create, modify, and delete individual pools`, is that sufficient?